### PR TITLE
Add Dime.Scheduler cloud connector

### PR DESCRIPTION
**Pull request by hbulens**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Dime.Scheduler/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,7878 @@
...
+        "text": {
+          "x-ms-summary": "Text",
+          "type": "string",
+          "description": "The notification message."
+        },
+        "date": {
+          "x-ms-summary": "Date",
+          "format": "date-time",
+          "type": "string",
+          "description": "Optional. The date stamp for this notificiation."
```
Comment (vmanoharas) [inline certified-connectors/Dime.Scheduler/apiDefinition.swagger.json]: Please correct the spelling

**Review by vmanoharas — COMMENTED** [review]
Hello @hbulens.

Swagger validation is failing, kindly sync the repo with main repo. Then, reopen the PR again with latest commit with changes.  Also please fix review comments.

**Review by vmanoharas — APPROVED** [review]
Hello @hbulens 

I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Comment by hbulens** [comment]
I tried to get the message across on ISV Studio, but it's impossible to proceed there. On submitting the connector files for certification, I just get a blank page.

image

image

**Comment by vmanoharas** [comment]
Hello @hbulens, Did you try uploading zip file instead commit id? if not can you please upload it as zip file.  If it doesn't work.  Can you please attach your artifacts in the ISV studio activity control.

**Comment by hbulens** [comment]
None of the files that I need to upload (.zip, .json), are accepted in ISV studio's activity control, so we'll have to find another way?

**Comment by vmanoharas** [comment]
Hi @hbulens,
Please send me your artifacts in a zip file format that you tried to upload via email.  Also please paste your error screen shot.

**Comment by hbulens** [comment]
See above for screenshots of the error.

[Dime.Scheduler.zip](https://github.com/microsoft/PowerPlatformConnectors/files/13547803/Dime.Scheduler.zip)

**Comment by vmanoharas** [comment]
Thank you @hbulens for the sharing the zip file.  I can see the screen shot, however I requested the error after you clicked on submit button, there should be some error message. However, we will investigate further and getting back to you in 2 days.

**Comment by hbulens** [comment]
There is no error message, unfortunately. The page just turns blank and throws the errors as shown in the console.

**Pull request merged**